src/protocolsupport/zplatform/impl/paper/entitytracker/PaperEntityTrackerEntry.java

```diff
@@ -0,0 +1,66 @@
+package protocolsupport.zplatform.impl.paper.entitytracker;
+
+import net.minecraft.server.v1_12_R1.Entity;
+import net.minecraft.server.v1_12_R1.EntityPlayer;
+import net.minecraft.server.v1_12_R1.EntityTracker;
+import net.minecraft.server.v1_12_R1.EntityTrackerEntry;
+import protocolsupport.api.ProtocolType;
+import protocolsupport.protocol.ConnectionImpl;
+import protocolsupport.utils.ReflectionUtils;
+import protocolsupport.zplatform.impl.spigot.entitytracker.SpigotEntityTrackerEntry;
+
+import java.lang.reflect.Field;
+import java.util.Map;
+
+public class PaperEntityTrackerEntry extends SpigotEntityTrackerEntry {
+	private static Field TRACKED_PLAYER_MAP_FIELD;
+	private static Field TRACKER_ENTITY_FIELD;
+	private Map<EntityPlayer, Boolean> trackedPlayerMap;
+
+	static {
+		try {
+			TRACKED_PLAYER_MAP_FIELD = EntityTrackerEntry.class.getDeclaredField("trackedPlayerMap");
```

Comment:
`ReflectionUtils.getField(EntityTrackerEntry.class, "trackedPlayerMap");`
setAccessible is then already done.